packages/suite/src/components/wallet/Fees/FeeDetails.tsx

```
@@ -1,120 +1,256 @@
+                                        symbol={symbol}
+                                        showApproximationIndicator
+                                    />
+                                }
+                                bottomRightChild={
+                                    <>
+                                        {formatFeePerUnit(fee?.feePerUnit)}{' '}
+                                        {getFeeUnits(networkType)}
+                                    </>
+                                }
+                            />
+                        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)